features/support/env.rb

```diff
@@ -1,9 +1,8 @@
 require 'aruba/cucumber'
-require "sinatra/base"
 
 Before do
   puts "Killing server..."
-  system "for i in `ps axu | grep 'server.rb' |  awk '{print $2}'`; do kill -9 $i; done > /dev/null 2>&1"
+  system "for i in `ps axu | grep 'server.js' |  awk '{print $2}'`; do kill -9 $i; done > /dev/null 2>&1"
```

Comment:
Would `pkill -9 node server.js` suffice?

Reply:
Perhaps, I'll be rewriting it in upcoming changes to address https://github.com/apiaryio/dredd-hooks-template/issues/6 as well.